packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -1,34 +1,34 @@
+ Client-based rate limiting applies to unauthenticated endpoints within Okta that are used during the access process for client applications. This applies to two types of client access:
 
 * A client accessing an OAuth 2.0 application, which uses a combination of client ID, IP address, and device identifier for rate-limiting.
 
 * A client accessing a non-OAuth 2.0 application (as is the case for a Classic Engine org `/login/login.htm` endpoint), which uses the IP address and device identifier for rate-limiting.
 
-The client access process can include requests to multiple API endpoints that differ based on your type of org: Okta Identity Engine or Okta Classic Engine.
+The client access process can include requests to multiple API endpoints that differ based on your type of org: Okta Identity Engine or Okta Classic Engine. The Identity Engine endpoints can include `/idp/idx`, `/idp/idx/identify`, `/idp/idx/introspect`, and [Identity Engine App intent links](/docs/concepts/oie-intro/#app-intent-links).
 
-The Classic Engine and the Identity Engine both include client-based rate-limiting for the OAuth 2.0 API endpoint `/authorize`. This endpoint provides granular isolation for requests made to the OAuth 2.0 `/authorize` endpoint, and this framework isolates rogue OAuth 2.0 clients and bad actors, thereby ensuring that valid users and applications don't run into rate limit violations.
+The Classic Engine and the Identity Engine both include client-based rate-limiting for the OAuth 2.0 API endpoint `/authorize`. This endpoint provides granular isolation for requests made to the OAuth 2.0 `/authorize` endpoint, and this framework isolates rogue OAuth 2.0 clients and bad actors, ensuring that valid users and applications don't run into rate limit violations.
 
 The Identity Engine includes client-based rate limiting for multiple API entry points that implement the [Interaction Code grant type](/docs/concepts/interaction-code/), which performs a series of interactions and API calls between the user and authorization server.
 
 The Classic Engine also includes client-based rate limiting for the `/login/login.htm` endpoint, and provides granular, targeted rate limits to a user, app, script, or server.
 
-The client-based rate limits for the OAuth 2.0 `/authorize` endpoint apply to both the Okta Org Authorization Server and any Custom Authorization Server. All Custom Authorization Servers share the same [rate limit](/docs/reference/rate-limits/). The Org Authorization Server has a separate rate limit.
+The client-based rate limits for the OAuth 2.0 `/authorize` endpoint apply to both the Okta org authorization server and any custom authorization server. All custom authorization servers share [rate limit](/docs/reference/rate-limits/). The org authorization server has a separate rate limit.
```

Comment:
Do all custom auth servers share the same limit? Or do they "share rate limits"?
```suggestion
The client-based rate limits for the OAuth 2.0 `/authorize` endpoint apply to both the Okta org authorization server and any custom authorization server. All custom authorization servers share [rate limits](/docs/reference/rate-limits/). The org authorization server has a separate rate limit.
```

Reply:
I'm fairly certain they share the same rate limit. Let me confirm.